### PR TITLE
Registration module 

### PR DESCRIPTION
**Pull request by cyberkitanoo**
## Issue that this pull request solves

Closes: #20 

## Proposed changes

This PR is to create a web-page to register new users

## Types of changes

_Put an `x` in the boxes replacing ' ' that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

image

## Other information

### CODE REVIEW

File: WebApp/dashboard/src/components/register.js [hunk 1]
```diff
@@ -4,10 +4,233 @@ class Register extends React.Component{
     render(){
         return(
             <div>
-                This is the register page
+                <!DOCTYPE html>
```
Comment (Saurabhdimri06) [inline WebApp/dashboard/src/components/register.js:7]: You need not to write whole HTML starting from Doctype in JS file

File: WebApp/dashboard/src/components/register.js [hunk 2]
```diff
@@ -4,10 +4,233 @@ class Register extends React.Component{
...
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```
Comment (Saurabhdimri06) [inline WebApp/dashboard/src/components/register.js:11]: For CSS there is a different file i.e. APP.JS or you can create a whole new file for the css of registration page

File: WebApp/dashboard/src/components/register.js [hunk 3]
```diff
@@ -4,10 +4,233 @@ class Register extends React.Component{
...
+    <div class="container" style="background-color:#f1f1f1">
+      <button type="button" onclick="document.getElementById('id01').style.display='none'" class="cancelbtn">Cancel</button>
+      
+    </div>
+  </form>
+</div>
+
+<script>
+// Get the modal
+var modal = document.getElementById('id01');
```
Comment (Saurabhdimri06) [inline WebApp/dashboard/src/components/register.js:218]: It is already a js framework so you need not to add script tag and define script it's better to define the onClick method before render -> Check on how to add onclick in react

File: screenshot/README.md [hunk 4]
```diff
@@ -1,3 +1,2 @@
-# ScreenShot Folder
-
-Complete the requisites of task 1 created and add the screenshot of github pages in this folder
+![](https://raw.githubusercontent.com/khuranagarvit019/retail_sale_prediction/master/khuranagarvit019.png)
```
Comment (Saurabhdimri06) [inline screenshot/README.md:1]: Why do we need a Readme File for register page?
Does it seem useful to you?

File: WebApp/dashboard/src/components/register.js (hunk 1 above)
Comment (cyberkitanoo) [reply WebApp/dashboard/src/components/register.js:7]: okay sir got it

**Pull request closed without merging**